Lib/test/test_signal.py

```diff
@@ -60,6 +60,10 @@ def test_getsignal(self):
     def test_strsignal(self):
         self.assertIn("Interrupt", signal.strsignal(signal.SIGINT))
         self.assertIn("Terminated", signal.strsignal(signal.SIGTERM))
+        if sys.platform.startswith("hp-ux"):
```

Comment:
I think `SIGHUP` is a pretty common signal, so it can be added to the list in `Modules/signalmodule.c`:

```c
#ifndef MS_WINDOWS
         /* TODO: Add a comment to explain why we need this on HP-UX. */
         case SIGHUP:
             res = "Hangup";
             break;
#endif
```

Then we can test `self.assertIn("Hangup", signal.strsignal(singal.SIGHUP))` unconditionally.